octodns/record.py

```diff
@@ -207,9 +207,15 @@ def validate(cls, name, data):
         values = []
         try:
             values = data['values']
+            if values is None or len(values) == 0:
+                values = []
+                reasons.append('missing value(s)')
```

Comment:
So the `values is None` is specifically for NSOne, as I believe that's the only case where a provider allows records with no answers. You end up with `data['values'] = None`

As the individual answers, I do think it's a good point to check them individually check to invalidate bad answers

If we do something like
```
test:
  type: A
  values:
    - ''
    -
```

We would get
```
octodns.record.ValidationError: Invalid record test.yzguy.io.
  - empty value
  - empty value
```

Which is good, because neither are good.